www/include/configuration/configObject/traps/formTraps.php

```diff
@@ -380,10 +380,8 @@ function myReplace()
 $form->addRule('traps_oid', _("Compulsory Name"), 'required');
 $form->addRule('manufacturer_id', _("Compulsory Name"), 'required');
 $form->addRule('traps_args', _("Compulsory Name"), 'required');
-$form->registerRule('exist', 'callback', [$trapObj, "testTrapExistence"]);
 $form->registerRule('wellFormated', 'callback', [$trapObj, "testOidFormat"]);
 $form->addRule('traps_oid', _("Bad OID Format"), 'wellFormated');
-$form->addRule('traps_oid', _("The same OID element already exists"), 'exist');
```

Comment:
`The same OID element already exists` should be removed from translation files

Reply:
Done